packages/react-playground/src/App.js

```diff
@@ -37,7 +39,17 @@ class App extends React.Component {
     this.setState({ disabled: !this.state.disabled });
   }
 
+  setSize(event) {
+    this.setState({ size: event.target.value });
```

Comment:
I see what you mean now. I was thinking in terms of how a native element would work (`element.value`). I don't think we should de-normalize our state object by setting a value property. I'll go with `state.options[state.selectedIndex]` in the sandbox.